src/components/inputs/z-input/index.stories.mdx

```diff
@@ -1,89 +1,320 @@
 import { html } from "lit-html";
-import { Meta, Story, Preview, Props } from "@storybook/addon-docs";
-import { text, boolean, number, radios, select } from '@storybook/addon-knobs';
-import { InputStatusEnum } from "../../../beans";
+import { Meta, Canvas, Story, Preview, Props } from "@storybook/addon-docs";
+import {
+  InputTypeEnum,
+  InputStatusEnum,
+  LabelPositionEnum,
+} from "../../../beans";
 import "./index";
 
 ## Stories
 
-<Meta title="Inputs/ZInput" component="z-input" />
+<Meta
+  title="Inputs/ZInput"
+  component="z-input"
+  argTypes={{
+    type: {
+      control: { type: "select" },
+      options: Object.values(InputTypeEnum),
+      defaultValue: "text",
```

Comment:
in console comunque vedo questo warning `argType.defaultValue is deprecated and will be removed in Storybook 7.0`

Reply:
Ouch

Reply:
😞

Reply:
A dire il vero nella documentazione della 7 https://storybook.js.org/docs/7.0/react/api/argtypes c'è ancora `defaultValue`. Per il momento lo lascerei, visto che non ho nemmeno trovato un valido sostituto. Che dici?

Reply:
che maledetti... ok allora quelli che non danno problemi lasciali pure, tanto immagino che passando alla 7 non saranno solo i `defaultValue` la cosa da modificare